apps/condo/domains/property/utils/testSchema/factories.js

```diff
@@ -96,7 +97,65 @@ const buildFakeAddressAndMeta = (withFlat, addressMetaExtraAttrs = {}) => {
     return { address, addressMeta }
 }
 
+const buildPropertyMap = ({ floors = 3, unitsOnFloor = 4, parkingFloors = 2, unitsOnParkingFloor = 8 } = {}) => ({
+    dv: 1,
+    type: 'building',
+    sections: [
+        {
+            id: '1',
+            type: SECTION_SECTION_TYPE,
+            index: 1,
+            name: '1',
+            preview: null,
+            floors: map(Array(floors), (...[, floor]) => {
+                const floorPlus1 = floor + 1
+                return {
+                    id: String(floorPlus1),
+                    type: 'floor',
+                    index: floorPlus1,
+                    name: String(floorPlus1),
+                    units: map(Array(unitsOnFloor), (...[, n]) => ({
+                        id: String(floor * unitsOnFloor + n + 1),
+                        type: 'unit',
+                        name: null,
+                        label: String(floor * unitsOnFloor + n + 1),
+                        preview: null,
+                        unitType: FLAT_UNIT_TYPE,
+                    })),
+                }
+            }),
+        },
+    ],
+    parking: [
+        {
+            id: '1',
+            type: SECTION_SECTION_TYPE,
+            index: 1,
+            name: '1',
+            preview: null,
+            floors: map(Array(parkingFloors), (...[, floor]) => {
+                const floorPlus1 = floor + 1
+                return {
+                    id: String(floorPlus1),
+                    type: 'floor',
+                    index: floorPlus1,
+                    name: String(floorPlus1),
+                    units: map(Array(unitsOnParkingFloor), (...[, n]) => ({
+                        id: String(floor * unitsOnParkingFloor + n + 1),
+                        type: 'unit',
```

Comment:
You can reuse unittype from constants

Reply:
Done. But this is not the same unitType as `flat`